src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -3672,14 +3673,18 @@ public int garbageCollect(String tombstoneOptionString, int jobs, String keyspac
     public void takeSnapshot(String tag, Map<String, String> options, String... entities) throws IOException
```

Comment:
@fibersel @pauloricardomg maybe this should not throw IOException anymore in case ttl is lower than 1 minute but I am not sure we can just change the signature of this method like that ...

Reply:
If we throw IllegalArgumentException, that is runtime exception so all good ...

Reply:
@smiklosovic can you resolve this?